src/site/layouts/press_release.drupal.liquid

```diff
@@ -0,0 +1,161 @@
+                <section class="vads-u-margin-bottom--5">
+                    <h1>{{ title }}</h1>
+                    <p class="vads-u-margin-bottom--1">PRESS RELEASE</p>
+                    <p class="vads-u-font-weight--bold vads-u-margin-bottom--4 vads-u-margin-top--0">{{ fieldReleaseDate.value | formatDate: 'MMMM D, YYYY' }}</p>
+                    <div>
+                        <a href="#" class="vads-u-margin-right--4" onclick="window.print(); return false;"><i class="fa fa-print"></i> Print</a>
+                        {% if fieldPdfVersion != empty %}
+                            <a href="{{ fieldPdfVersion.entity.fieldDocument.entity.url }}" download><i class="fa fa-download"></i> Download a PDF version</a>
+                        {% endif %}
+                    </div>
+                    <p class="va-introtext">{{ fieldAddress.locality }}, {{ fieldAddress.administrativeArea }} — {{ fieldIntroText }}</p>
+                    <div>{{ fieldPressReleaseFulltext.processed }}</div>
+                </section>
+
+                {% if fieldPressReleaseContact != empty %}
+                    <section class="vads-u-margin-bottom--5">
+                        <div class="vads-u-font-weight--bold">Media contacts</div>
+                        {% for contact in fieldPressReleaseContact %}
+                            {% assign c = contact.entity %}
+                            <div>
+                                <p>{{ c.title }}{% if c.fieldDescription != empty %}, {{ c.fieldDescription }} {% endif %}</p>
+                                <p>{{ c.fieldPhoneNumber }}</p>
+                                <p><a href="mailto:{{ c.fieldEmailAddress }}">{{ c.fieldEmailAddress }}</a></p>
+                            </div>
+                        {% endfor %}
+                    </section>
+                {%  endif %}
+
+                {% if fieldPressReleaseDownloads != empty %}
+                    <section class="vads-u-margin-bottom--5">
+                        <div class="vads-u-font-weight--bold vads-u-margin-bottom--1p5">Download media assets</div>
+                        {% for asset in fieldPressReleaseDownloads %}
+                            {% assign a = asset.entity %}
+                            <ul class="vads-u-margin-bottom--1 usa-unstyled-list">
+                                <li>
+                                    {% case a.entityBundle %}
+                                        {% when 'document' %}
+                                            <a href="{{ a.fieldDocument.entity.url }}" download>{{ a.name }}</a>
+                                        {% when a.entityBundle === 'image' %}
+                                            <a href="{{ a.image.url }}" download>{{ a.name }}</a>
+                                        {% when a.entityBundle === 'video' %}
+                                            <a href="{{ a.fieldMediaVideoEmbedField }}" target="_blank">{{ a.name }}</a>
```

Comment:
You can remove the `target` attribute here, it'll get added if necessary based on the url in the href.

Reply:
ok, removing